### DISCUSSION TITLE
How Do I Enable A Custom Style?

### DISCUSSION THREAD

**Discussion by VehementHam** [pygments/pygments#2718] [Q&A]
How do I enable a custom style so that when I use pygments, it uses that style by default?

Say I run `pygmentize -g`, I want that to use the set pygments style.

Here is the style that I want to install: https://github.com/mozmorris/tomorrow-pygments

**Comment by tanviranindo** [comment]
Here is a reliable way to use a third-party style (like tomorrow-pygments) with Pygments and `pygmentize`.

### 1. Install the style package

If it is on PyPI, prefer `pip install <package-name>`. Otherwise install from the repo (example):

```bash
pip install "tomorrow-pygments @ git+https://github.com/mozmorris/tomorrow-pygments.git"
```

### 2. Confirm Pygments sees the style

```bash
pygmentize -L styles
```

Use the **exact** style name from that list (for example a name containing `Tomorrow` — the registered name is defined by the style package).

### 3. Use it with `pygmentize`

```bash
pygmentize -g -O style=<ExactNameFromStep2> yourfile.py
```

### 4. Making it the default for every run

`pygmentize` does not always expose a single global user config for default style in every setup. Practical options:

- **Shell alias**, e.g. `alias pygmentize="pygmentize -O style=TomorrowNight"` (use your real style name).
- A small **wrapper script** that injects `-O style=...` when you omit it.

### If the style does not appear in `pygmentize -L styles`

The package must register a Pygments style (typically a `pygments.styles` entry point in `pyproject.toml` / `setup.cfg`). If it still does not show up after install, that is an issue for the style repository maintainers.

If you paste the output of listing styles filtered for tomorrow (or say whether `pip show tomorrow-pygments` reports an installed package), we can pin down the exact `style=` string.
